dev/com.ibm.ws.cdi.visibility_fat/.classpath

```diff
@@ -17,7 +17,7 @@
 	<classpathentry kind="src" path="test-applications/warLibAccessBeansInWarJar.jar/src"/>
 	<classpathentry kind="src" path="test-applications/TestValidatorInJar.jar/src"/>
 	<classpathentry kind="src" path="test-applications/TestValidatorInJar.war/src"/>
-	<classpathentry kind="src" path="test-applications/jarInRar.rar/src"/>
+	<classpathentry kind="src" path="test-applications/jarInRar.rar/resources"/>
```

Comment:
Since there is no java source in this directory, I didn't think we included it in .classpath.  At least other projects that have 'resources' directories with just XML have not done this.